grunt/karma.js

```diff
@@ -17,6 +17,9 @@ module.exports = function (grunt) {
                     '<%= config.lib %>/angular-ext-q/angular-ext-q.js',
                     '<%= config.lib %>/angular-mocks/angular-mocks.js',
 
+                    '<%= config.lib %>/jquery/dist/jquery.js',
```

Comment:
`jquery` is marked as a `devDependency` in `bower.json`. This line means that it's required, not just a dev-specific thing (like `angular-mocks`). Should it be a `dependency` instead?